homework-1/index.html

```diff
@@ -0,0 +1,681 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Scout</title>
+  <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css" />
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div id="banner-submenu" class="fixed-top">
+    <nav class="submenu">
+      <ul class="submenu__list list-top" id="list-top">
+        <li class="first-item"><a class="tips-active" href="#tips">Tips</a></li>
+        <li class="submenu__item-middle"><a class="submenu__link" href="#glossary">Glossary</a></li>
+        <li class="last-item"><a href="#fqas">FAQs</a></li>
+      </ul>
+    </nav>
+  </div>
+  <header class="header" id="header">
+    <div class="logo">
+      <a class="logo__link" href="#">
+        skout
+      </a>
+    </div>
+    <div class="header__menu menu">
+      <div class="menu__icon">
+        <span></span>
+      </div>
+      <nav class="menu__body">
+        <ul class="menu__list">
+          <li class="link-hover-center"><a class="menu__link first-link" href="#">Why Skout </a></li>
+          <li class="link-hover-center"><a class="menu__link" href="#">Products</a></li>
+          <li class="link-hover-center"><a class="menu__link" href="#">Partners</a></li>
+          <li class="link-hover-center"><a class="menu__link" href="#">Company</a></li>
+          <li class="link-hover-center active"><a class="menu__link" href="#">Tips & Tools</a></li>
+          <li class="link-hover-center"><a class="menu__link" href="#">Blog</a></li>
+          <div class="header__button-link">
```

Comment:
Please move this block from here. At least you brake semantics.

Reply:
fixed a bug